cypress/TestCases/FireboltCertification/Localization.feature

```diff
@@ -88,27 +89,24 @@ Feature: Localization
         And 'Firebolt' platform triggers '3rd party app' event
 
         Examples:
-            | Scenario                                   | Method                  | Value      |
-            | Set & get locality                         | locality                | washington |
-            | Set & get countrycode                      | countryCode             | PH         |
-            | Set & get locale                           | locale                  | enUK       |
-            | Set & get Language es                      | language                | es         |
-            | Set & get Language en                      | language                | en         |
-            | Set & get preferredAudioLanguages(spa-eng) | preferredAudioLanguages | spa,eng    |
-            | Set & get preferredAudioLanguages(eng-spa) | preferredAudioLanguages | eng,spa    |
-            | Set & get PostalCode                       | postalCode              | "12345"    |
+            | Scenario      | Method                  | Value      |
+            | washington    | locality                | washington |
+            | PH            | countryCode             | PH         |
+            | Spanish       | preferredAudioLanguages | spa,eng    |
+            | English       | preferredAudioLanguages | eng,spa    |
+            | 12345         | postalCode              | 12345      |
```

Comment:
In line 97, we have to give  "12345" . Otherwise it would fail

Reply:
Fixed value for postal code and changed to string.